ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -773,8 +826,12 @@ S3Flush(WT_STORAGE_SOURCE *storageSource, WT_SESSION *session, WT_FILE_SYSTEM *f
     bool nativeExist;
     FS2S3(fileSystem)->statistics.putObjectCount++;
 
+    s3->log->LogDebugMessage("S3Flush: Flush to S3 Store using AWS SDK C++ PutObject");
```

Comment:
np: Not necessary to say that you are using the AWS SDK C++ as this is used all throughout and is implied.

Reply:
Yeah I added this for debugging purposes but it's not necessary, I've removed it.